db/model/version.go

```diff
@@ -43,7 +43,7 @@ type VersionInfo struct {
 	RepoURL       string `gorm:"column:repo_url;size:2047" json:"repo_url"`
 	CodeVersion   string `gorm:"column:code_version;size:40" json:"code_version"`
 	CodeBranch    string `gorm:"column:code_branch;size:40" json:"code_branch"`
-	CommitMsg     string `gorm:"column:code_commit_msg;size:200" json:"code_commit_msg"`
+	CommitMsg     string `gorm:"column:code_commit_msg;size:65535" json:"code_commit_msg"`
```

Comment:
This is not the ultimate solution, but can be defined as up to 1024, preferably truncating the excess information before storing it.